### PR TITLE
add pentapy recipe

### PR DESCRIPTION
**Pull request by MuellerSeb**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by MuellerSeb** [comment]
What is happening on appveyor? There seems to be a problem with `cl.exe`?

**Comment by MuellerSeb** [comment]
@conda-forge-admin, please restart ci

**Comment by MuellerSeb** [comment]
Analogous to #11262 , the appveyor failure can be ignored.
Ready for review.

**Comment by MuellerSeb** [comment]
@conda-forge-admin, please restart ci

**Comment by MuellerSeb** [comment]
@conda-forge-admin, please ping team

File: recipes/pentapy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a242f06ce9b2a806e544dbbbb497c3ae541e76d366114564cf877cb8b82f0bfd
+
+build:
+  number: 0
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
```
Comment (xhochy) [inline recipes/pentapy/meta.yaml:15]: Normally, this shouldn't be needed as `setuptools_scm` bakes in the version into the sdist.

Comment (MuellerSeb) [reply recipes/pentapy/meta.yaml:15]: ok, I'll remove the pretended version, but it shouldn't harm the building.

**Comment by MuellerSeb** [comment]
@conda-forge-admin, please restart ci

**Pull request merged**